### PR TITLE
[MOV] purchase: moving doc page from accounting to purchase

### PR DESCRIPTION
**Pull request by Donapi**
Moving the Manage vendor bills page from the accounting app to the purchase app as it is more relevant.

taskid: 2687652

### CODE REVIEW

**Comment by Donapi** [comment]
Hi @brse-odoo and @larm-odoo 
I created this PR following our discussion to move the Manage vendor bills page from the accounting section to the purchase section as it is more relevant. The page is no longer up to date and needs to be worked on.
Thanks in advance :)

**Review by brse-odoo — APPROVED** [review]
Hey @Donapi! Structure looks good, I don't have any changes to add for the structure. I agree that this is more relevant under Purchase! As far as updating the content of the doc, we will open a new PR to improve the content.

**Comment by brse-odoo** [comment]
Hi @StraubCreative, this PR is ready for your review. Thank you!

**Comment by StraubCreative** [comment]
Looks good! Thanks for your work on this @Donapi we'll update everything in another PR.
@odoo/doc-review this accounting --> purchase file move is ready for your review.

File: content/applications/finance/accounting/payables/pay/multiple.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Odoo offers the possibility of grouping multiple bills' payments into one, facil
 reconciliation process.
 
 .. seealso::
-   - :doc:`../supplier_bills/manage`
+   - :doc:`../../../../inventory_and_mrp/purchase/manage_deals/manage`
```
Comment (Feyensv) [inline content/applications/finance/accounting/payables/pay/multiple.rst:9]:
```suggestion
   - :doc:`/applications/inventory_and_mrp/purchase/manage_deals/manage`
```
You can start references from the `content` folder when it's a shorter and more readable path ;).

**Review by Feyensv — COMMENTED** [review]
It seems you are duplicating the images without removing the old ones.
Are they used by another page ?

**Comment by StraubCreative** [comment]
> It seems you are duplicating the images without removing the old ones. Are they used by another page ?

Ah, good catch sorry I missed that.
@samueljlieber can you `git rm` the old images + folder for this doc that was in the accounting module and fix `:doc:` paths so they start with `/applications`

**Comment by samueljlieber** [comment]
https://github.com/odoo/documentation/commit/6e78067a6feeaa83c9fbeb0e58e811026e15b541 updated `:doc:` paths and `git rm`'d the images 😁

CC @StraubCreative

**Comment by StraubCreative** [comment]
Should be good for another look @Feyensv 👍 
Thank you.

File: redirects/14.0.txt [hunk 2]
```diff
@@ -15,6 +15,9 @@ applications/sales/crm/acquire_leads/generate_from_website.rst applications/sale
 
 developer/misc/api/external_api.rst developer/api/external_api.rst  # better location for misc/ content (#2350)
 
+# finance
+finance/accounting/payables/supplier_bills/manage.rst applications/inventory_and_mrp/purchase/manage_deals/manage.rst # (#3302)
```
Comment (Feyensv) [inline redirects/14.0.txt:19]:
```suggestion
applications/finance/accounting/payables/supplier_bills/manage.rst applications/inventory_and_mrp/purchase/manage_deals/manage.rst # (#3302)
```

**Review by Feyensv — COMMENTED** [review]
Little point I missed and it's good to go :)

@robodoo delegate+
@robodoo delegate=StraubCreative

**Comment by StraubCreative** [comment]
Fixed redirect on [5ec9a1a](https://github.com/odoo/documentation/pull/3302/commits/5ec9a1a192b76101d694caf2f6919cf006201e44). Checks pass 👍 

Thanks for the review @Feyensv

@robodoo r+

**Pull request closed without merging**